packages/workbox-routing/src/Router.ts

```diff
@@ -264,30 +271,12 @@ class Router {
    *     They are populated if a matching route was found or `undefined`
    *     otherwise.
    */
-  findMatchingRoute({url, request, event}: {
-    url: URL;
-    request: Request;
-    event?: ExtendableEvent;
-  }): {route?: Route; params?: HandlerCallbackOptions['params']} {
-    if (process.env.NODE_ENV !== 'production') {
-      assert!.isInstance(url, URL, {
-        moduleName: 'workbox-routing',
-        className: 'Router',
-        funcName: 'findMatchingRoute',
-        paramName: 'options.url',
-      });
-      assert!.isInstance(request, Request, {
-        moduleName: 'workbox-routing',
-        className: 'Router',
-        funcName: 'findMatchingRoute',
-        paramName: 'options.request',
-      });
-    }
-
+  findMatchingRoute({url, sameOrigin, request, event}: MatchCallbackOptions):
+      {route?: Route; params?: HandlerCallbackOptions['params']} {
```

Comment:
I think we agreed that the dev-only `assert!.isInstance()`-style checks for parameters was less useful now that folks are migrating towards TypeScript, so instead of adding a check here for `sameOrigin`, I remove the existing checks.

Reply:
SGTM

Reply:
SGTM